Sources/TootSDK/Models/TootNotificationParams.swift

```diff
@@ -19,12 +22,28 @@ public struct TootNotificationParams: Codable, Sendable {
     }
 
     /// Types of notifications to exclude from the search results
-    public var excludeTypes: [TootNotification.NotificationType]?
+    public var excludeTypes: Set<TootNotification.NotificationType>?
     /// Types of notifications to include in the search results
-    public var types: [TootNotification.NotificationType]?
+    public var types: Set<TootNotification.NotificationType>?
 
     enum CodingKeys: String, CodingKey {
         case excludeTypes = "exclude_types"
         case types = "types"
     }
 }
+
+extension TootNotificationParams {
+    func corrected(for flavour: TootSDKFlavour) -> TootNotificationParams {
```

Comment:
```suggestion
    private func adapted(for flavour: TootSDKFlavour) -> TootNotificationParams {
```

Reply:
It is inheriting its default internal visibility which is needed to call it from other files in module. I can add explicit `internal` if you prefer.